posthog/api/team.py

```diff
@@ -90,6 +92,18 @@ def get_effective_membership_level(self, team: Team) -> Optional[OrganizationMem
     def get_has_group_types(self, team: Team) -> bool:
         return GroupTypeMapping.objects.filter(team=team).exists()
 
+    def get_event_first_seen(self, team: Team) -> Optional[datetime.datetime]:
+        # TODO: Remove try statement when CH is moved out ee/
+        try:
+            from ee.clickhouse.client import sync_execute
+        except ImportError:
+            return None
+
+        response = sync_execute(
```

Comment:
Seems like an innocuous enough query as a) it's very light, and b) we only use this serializer when retrieving the current project (i.e. `/api/projects/{id}`) not on project list or nested serializers, but open to challenge on whether this should be computed async.

Reply:
Need to think on this - we already are doing a lot of unneeded clickhouse load every pageload, and this smells like the same. Will think on this.

Reply:
For Q2, it shouldn't. If CH is down, we will just return `None` which should be interpreted as unable to deetermine.